test/metabase/models/dashboard_test.clj

```diff
@@ -3,23 +3,16 @@
    [clojure.test :refer :all]
    [metabase.api.common :as api]
    [metabase.automagic-dashboards.core :as magic]
-   [metabase.models.card :refer [Card]]
```

Comment:
this is here because I cleaned the up ns while adding test to this ns. but the test was removed but I think it's still good to leave this.